### ISSUE TITLE
Doesn't allow switching Google Drive account

### ISSUE THREAD

**Issue by benvmatheson** [perfectapi/CKP#122]
I have two Google drive accounts tied together. When selecting the database, CKP always attempts to use my second account, instead of the account where the database is stored.

**Comment by benvmatheson** [comment]
I can make this work by sharing the file with my second account, but I would rather not have my password database stored on my second account.

**Comment by nemoyatpeace** [comment]
Same issue for me. :(

**Comment by DukeyToo** [comment]
When this occurs, are you logged into Chrome with the other account?  I think it probably uses whatever account is logged into Chrome by default.

**Comment by nemoyatpeace** [comment]
Yes, I am logged in to chrome with the account it is grabbing. I'm not
logged into any Google services. I don't want to log out of chrome though.

**Comment by benvmatheson** [comment]
The same is true for me. I think the problem is that I'm logged in with both accounts, so it doesn't allow me to specify which to use. Generally I can change the URL from 0 to 1 to switch accounts:

https://mail.google.com/mail/u/0/#inbox
https://mail.google.com/mail/u/1/#inbox

But I don't have that ability here.

**Comment by DukeyToo** [comment]
It looks like this should be possible - the API we're using internally is https://developer.chrome.com/apps/identity#method-getAuthToken and that method does support specifying an account id.  The flag `enable-new-profile-management` mentioned in the docs should be enabled by default in newer Chrome versions.
